v4/feature/trade/src/main/java/exchange/dydx/trading/feature/trade/targetleverage/DydxTradeInputTargetLeverageView.kt

```diff
@@ -55,14 +56,20 @@ object DydxTradeInputTargetLeverageView : DydxComponent {
         val localizer: LocalizerProtocol,
         val leverageText: String?,
         val leverageOptions: List<LeverageTextAndValue>?,
+        val logoUrl: String? = null,
         val selectAction: ((String) -> Unit)? = null,
         val closeAction: (() -> Unit)? = null,
     ) {
         companion object {
             val preview = ViewState(
                 localizer = MockLocalizer(),
                 leverageText = "1.0",
-                leverageOptions = listOf(),
+                leverageOptions = listOf(
+                    LeverageTextAndValue("1.0", "1.0"),
+                    LeverageTextAndValue("2.0", "2.0"),
+                    LeverageTextAndValue("5.0", "5.0"),
+                    LeverageTextAndValue("10.0", "10.0"),
+                ),
```

Comment:
i think max leverage can sometimes be 5? This is what I did in iOS 

https://github.com/dydxprotocol/v4-native-ios/blob/e89aaba312de14dd0f17f1ab9f9e26eaa46d20ad/dydx/dydxPresenters/dydxPresenters/_v4/Trade/Margin/dydxTargetLeverageViewBuilder.swift#L98

maybe overkill 🤷

Reply:
This is just for UI preview during development.